packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -23,6 +23,27 @@ export interface ClusterProps {
    * The VPC where your ECS instances will be running or your ENIs will be deployed
    */
   readonly vpc: ec2.IVpc;
+
+  /**
+   * The service discovery namespace created in this cluster
+   *
+   * @default none
```

Comment:
Use `-` as a default value and describe the behavior. Reference the method

Reply:
Addressed.